apps/03TicTacToe.js

```diff
@@ -20,25 +20,80 @@ function printBoard() {
     console.log('  ---------');
     console.log('2 ' + board[2].join(' | '));
 }
+//function getPrompt(){
+  //printBoard();
+  //prompt.get([row, column]),
+//}
 
 function horizontalWin() {
     // Your code here
+    if (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn){
```

Comment:
Not bad! we could actually just use || here, but good clean indentation & structure